### PR TITLE
Ivo/Misc Fixes

### PR DESCRIPTION
**Pull request by izivkov**
- Delay starting the RTSP client 1 second to improve staring issues.
- Move setting resolution from RtspServer to PhoneController
- Select Nearby connection when WiFi is turned off.

### CODE REVIEW

File: android/app/src/main/java/org/openbot/env/IVideoServer.java [hunk 1]
```diff
@@ -5,10 +5,10 @@
 import android.view.TextureView;
 
 public interface IVideoServer {
-  void startServer();
```
Comment (thias15) [inline android/app/src/main/java/org/openbot/env/IVideoServer.java:8]: Why is this no longer needed?

Comment (thias15) [reply android/app/src/main/java/org/openbot/env/IVideoServer.java:8]: I see, this seems to be handled now in RTSP server with `private void startServer` and is not part of the interface anymore.

**Review by thias15 — COMMENTED** [review]
Pending testing. I don't see how video is now supported in all fragments.

**Comment by izivkov** [comment]
> Pending testing. I don't see how video is now supported in all fragments

Yes, the name of the branch is misleading. This was my intent, but as I described in Slack, ran into problems with dynamically creating surfaces. The description of the MR is correct. I will see if I can change the name of the brarch.

File: android/app/src/main/java/org/openbot/env/IVideoServer.java (hunk 1 above)
Comment (izivkov) [reply android/app/src/main/java/org/openbot/env/IVideoServer.java:8]: > Why is this no longer needed?

The server is started implicitly now when all the input conditions are met. Take a look at how `AndGate` is used.

Comment (izivkov) [reply android/app/src/main/java/org/openbot/env/IVideoServer.java:8]: On second thought, maybe we should keep this in the interface. Looks like a method you would normally use. I think I will change it back.

**Comment by thias15** [comment]
Ok, makes sense. Will test and then merge.

**Review by thias15 — APPROVED** [review]

**Pull request merged**